2020-02-Spring/1_strings/README.md

```diff
@@ -124,13 +193,24 @@ Go to [Top](#top)
 Source: TODO :bug:
 
 #### Naive/Simple Solution
+<!-- Don't remove the tags below -->
 
+<details>
+<summary>Click to see solution</summary>
+ 
 TODO :bug:
 
+</details>
```

Comment:
_**Use great caution when modifying the other contributor's sections.**_ Either you or they will **need to resolve merge conflicts** before all changes can be pushed to master.

This is a good exercise, and I think the dropdowns add a lot of value, but not everyone who is contributing will have enough git acumen to resolve the conflicts, or feel comfortable doing so.

Reply:
I anticipated conflicts when I did this. Luckily I did a conflict resolve before (for Hackathon docs). It's going to be a good learning opportunity. If it works well, the future ctci template will implement this drop-down feature.